src/clients/host-client.js

```diff
@@ -324,6 +324,27 @@ class HostClient extends BaseEvernodeClient {
             memos,
             options.transactionOptions);
     }
+
+    async pruneInstance(txHash, tenantAddress, refundAmount, options = {}) {
```

Comment:
What if we change the method name to this "refundWhenInstancePrune" or something else that showcases the refund nature.